src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -680,55 +698,39 @@ export default {
 		// Checks whether the conversations list is scrolled all the way to the top
 		// or not
 		handleScroll() {
-			this.isScrolledToTop = this.$refs.container.scrollTop === 0
-		},
-		elementIsAboveViewpoint(container, element) {
-			return element.offsetTop < container.scrollTop
-		},
-		elementIsBelowViewpoint(container, element) {
-			return element.offsetTop + element.offsetHeight > container.scrollTop + container.clientHeight
+			this.isScrolledToTop = this.$refs.scroller.$el.scrollTop === 0
 		},
+
+		/**
+		 * Find position of unread conversation below viewport
+		 */
 		handleUnreadMention() {
-			this.unreadNum = 0
-			const unreadMentions = document.querySelectorAll('.unread-mention-conversation')
-			unreadMentions.forEach(x => {
-				if (this.elementIsBelowViewpoint(this.$refs.container, x)) {
-					if (this.unreadNum === 0) {
-						this.firstUnreadPos = x.offsetTop
+			this.$nextTick(() => {
+				this.unreadNum = 0
+				const lastConversationInViewport = this.$refs.scroller.getLastItemInViewportIndex()
+				for (let i = this.filteredConversationsList.length - 1; i > lastConversationInViewport; i--) {
```

Comment:
```suggestion
				for (let i = lastConversationInViewport; i < this.filteredConversationsList.length - 1; i++) {
```
I'd go with another order, as unread mentions have newer lastActivity, and stay higher in the list

Reply:
I kept the current algorithm, we scroll to the latest (most down) unread mention.

Actually, first I suddenly implemented it as you proposed :D 

And then there was a problem. If you have a long list of unread mentions, then, even if you scroll to the first one, you still have some unread mentions below the viewport.

So 
1. You click on "Unread mentions"
2. It scrolls to the beginning of the long list of unread mentions
3. But you still see the button
4. You click again and move just one move conversation down
5. And then again, until you move to the last one

Reply:
Is that a feature, like an inbox, to process all one by one?)

Reply:
Well, we have a filter already to have all unread now :D

So I think the primary feature of this button is to indicate that you have something unread below.